dashboards/grafana-dashboard-rhtap-slos.configmap.yaml

```diff
@@ -1179,117 +1104,72 @@ data:
               "refId": "A"
             }
           ],
-          "title": "% of Healthy Route",
+          "title": "Application Creation SLO",
           "type": "text"
         },
         {
           "datasource": {
             "type": "prometheus",
             "uid": "${datasource}"
           },
+          "description": "The current measurement for the given time window.",
           "fieldConfig": {
             "defaults": {
               "color": {
-                "mode": "palette-classic"
+                "mode": "thresholds"
               },
-              "custom": {
-                "axisCenteredZero": false,
-                "axisColorMode": "text",
-                "axisLabel": "Percentage",
-                "axisPlacement": "auto",
-                "barAlignment": 0,
-                "drawStyle": "line",
-                "fillOpacity": 0,
-                "gradientMode": "none",
-                "hideFrom": {
-                  "legend": false,
-                  "tooltip": false,
-                  "viz": false
-                },
-                "lineInterpolation": "linear",
-                "lineStyle": {
-                  "fill": "solid"
-                },
-                "lineWidth": 1,
-                "pointSize": 5,
-                "scaleDistribution": {
-                  "type": "linear"
-                },
-                "showPoints": "auto",
-                "spanNulls": false,
-                "stacking": {
-                  "group": "A",
-                  "mode": "none"
-                },
-                "thresholdsStyle": {
-                  "mode": "dashed+area"
+              "mappings": [
+                {
+                  "options": {
+                    "match": "null",
+                    "result": {
+                      "text": "N/A"
+                    }
+                  },
+                  "type": "special"
                 }
-              },
-              "mappings": [],
-              "max": 120,
+              ],
+              "max": 1,
               "min": 0,
               "thresholds": {
-                "mode": "absolute",
+                "mode": "percentage",
                 "steps": [
                   {
-                    "color": "red",
+                    "color": "green",
                     "value": null
                   },
                   {
-                    "color": "green",
+                    "color": "red",
                     "value": 95
-                  }
-                ]
```

Comment:
It looks like the colors should be reversed here.

![image](https://github.com/redhat-appstudio/o11y/assets/22659204/f698ca24-0fc3-4525-b750-79f77a3adef6)

Reply:
Fixed:  
<img width="983" alt="image" src="https://github.com/redhat-appstudio/o11y/assets/84398375/2fe26daa-0f50-4405-a6d4-6469c07ddeb5">